webui/src/lib/components/controls.jsx

```diff
@@ -85,7 +85,7 @@ export const Error = ({error, onDismiss = null, className = null}) => {
         return <Alert className={className} variant="danger" dismissible onClose={onDismiss}>{content}</Alert>;
     }
     if (err.stack) {
-        content = `${content}: ${err.stack}`
+        content = `${content}`
     }
```

Comment:
This block is redundant now, you can remove it

Reply:
Right! 🤦🏻‍♂️
Fixed.